transformations/diverse_paraphrase/submod/submodular_funcs.py

```diff
@@ -0,0 +1,218 @@
+import numpy as np
+import scipy.linalg as la
+import pdb
+from nltk import ngrams
+import difflib
+import pickle
+from time import time
+import os
+import torch
+
+import urllib.request
+from tqdm import tqdm
+from scipy.spatial.distance import pdist, squareform
+import scipy
+from numpy import dot
+from numpy.linalg import norm
+import gzip
+import urllib
+from torchtext.vocab import GloVe
+from pathlib import Path
+
+
+model = None
+
+
+def trigger_dips():
+    global model
+    def unk_init(x):
+        return torch.randn_like(x)
+    model = GloVe('6B', unk_init=unk_init)
```

Comment:
A lot of things seem to be consuming memory here - this can be reduced to 50 dim instead of the default 300.

Reply:
Updated!